main.js

```diff
@@ -0,0 +1,76 @@
+// // mobile menu
+const openMenu = document.querySelector('.menubar');
+const closeMenu = document.querySelector('.close-x');
+const mobileMenu = document.querySelector('.navlist1');
+
+openMenu.addEventListener('click', () => {
+  mobileMenu.style.display = 'flex';
+});
+
+closeMenu.addEventListener('click', () => {
+  mobileMenu.style.display = 'none';
+});
+
+// Object Featured speaker
+
+const cardList = document.getElementById('pop');
+
+const projectsList = [
```

Comment:
- [ ] [REQUIRED] As shown in the design 🖼️, Please add 2 more speakers to the list of speakers since the design shows **6** Speakers ➕.

Reply:
Done